File: Pertemuan 5/PredikatKelulusan.java
// Tugas Praktek Pertemuan 5
import java.util.Scanner;

public class PredikatKelulusan{
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukan ipk Anda = ");
        double ipk = scanner.nextDouble();

        System.out.print("Masukan lama masa studi = ");
        double lulusTahun = scanner.nextDouble();

        if (ipk >= 3.51 && lulusTahun <= 4){
            System.out.println("Predikat anda Summa Cumlaude");
        } else if (ipk >= 3.51){
            System.out.println("Predikat anda Cumlaude");
        } else if (ipk >= 3.01 && ipk < 3.51){
            System.out.println("Predikat anda Sangat Memuaskan");
        } else if (ipk >= 2.76 && ipk < 3.01){
            System.out.println("Predikat anda Memuaskan");
        } else if (ipk >= 2.00 && ipk < 2.76){
            System.out.println("Predikat anda Cukup");
        } else if (ipk < 2.00){
            System.out.println("Anda Tidak Lulus");
        }
        System.out.println("Anda Tidak Valid");

    }
}

File: Pertemuan 5/predikatkelulusan.py
# Tugas Praktek Pertemuan 5
ipk = input("Masukan ipk Anda = ")
ipk = float(ipk)

lulusTahun = input("Masukan lama masa studi = ")
lulusTahun = str(lulusTahun)

if (ipk > 3.51 and lulusTahun <= str(4)):
    print("Predikat anda Summa Cumlaude")
elif (ipk >= 3.51):
    print("Predikat anda Cumlaude")
elif (ipk >= 3.01 and ipk < 3.51):
    print("Predikat anda Sangat Memuaskan")
elif (ipk >= 2.76 and ipk < 3.01):
    print("Predikat anda Memuaskan")
elif (ipk >= 2.00 and ipk < 2.76):
    print("Predikat anda Cukup")
elif (ipk < 2.00):
    print("Anda Tidak Lulus")
else:
    print("Anda Tidak Valid")
